.github/workflows/build.yml

```diff
@@ -156,6 +156,7 @@ jobs:
             os: macos-13
           - name: macos-arm64
             os: macos-14
+            config_arg: CONFIG_ARG='--enable-codegen-invariants'
```

Comment:
GitHub is busy allowing issues to be grouped in trees, rather than useful features like arbitrary code suggestions in PRs, but rather than this I'd...
```suggestion
```
and then in the "configure tree" bit below do:
```diff
        run: |
-          CONFIG_ARG='${{ matrix.config_arg }}' MAKE_ARG=-j bash -xe tools/ci/actions/runner.sh configure
+          CONFIG_ARG='--enable-codegen-invariants ${{ matrix.config_arg }}' MAKE_ARG=-j bash -xe tools/ci/actions/runner.sh configure
```